### DISCUSSION TITLE
swarp stars?

### DISCUSSION THREAD

**Discussion by famc2** [phoebe-project/phoebe2#768] [Questions]
after estimate, I think t0_supconj is half peroid ahead.
How to exchange starA and starB? will exchange slove the problem?

![lc](https://github.com/phoebe-project/phoebe2/assets/16513746/6597e56e-d772-4ec1-9e8e-862265d5f574)

THANKS!

**Comment by kecnry** [comment]
Assuming you're talking about the lc geometry estimator, you can try setting `t0_supconj` to near where you think it should be and re-running the estimator.  If that still doesn't give you the results you'd expect, you can either not adopt `t0_supconj` or override it after-the-fact.  There is no method for swapping the roles of the stars directly (technically you could do so via the hierarchy parameter, but I don't think its worth it for a case like this), but you can always manually swap the values of whichever parameters you think necessary. 

Hope this helps!
-Kyle
